views/mdc/assets/js/utils/drag_n_drop.js

```diff
@@ -0,0 +1,74 @@
+export const DnD = function () {
+    var initDraggables = function (selector) {
+            let draggables = document.querySelectorAll(selector);
+            [].forEach.call(draggables, function (elem) {
```

Comment:
May want to modernize this a bit (for consistency) since we're running everything through Babel.

```suggestion
            for (const elem of draggables) {
```

Reply:
Took you recommendations and modernized/cleaned up the JS.
Fixed FF redirect issue when dropping
Specs now succeeding.